tasks/connectors/veracode_asset_vulns/lib/veracode_av_client.rb

```diff
@@ -256,6 +257,7 @@ def get_findings_sca(app_guid, app_name, tags, page_size)
               cwe = finding["finding_details"]["cwe"].fetch("id") if finding["finding_details"]["cwe"]
               cwe = "CWE-#{cwe}" if finding["finding_details"]["cwe"]
               cve = finding["finding_details"]["cve"].fetch("name").strip if finding["finding_details"]["cve"]
+              cve_sol = finding["finding_details"]["cve"]["href"] if finding["finding_details"]["cve"]["href"]
```

Comment:
What is "cve_sol"?  "cve_solution"?   Could the variable name be more complete?

Reply:
Thanks. Done